NetKet/Machine/conv_layer.hpp

```diff
@@ -46,7 +46,6 @@ class Convolutional : public AbstractLayer<T> {
   int out_channels_;  // number of output channels
   int out_size_;      // output size: should be multiple of no. of sites
   int npar_;          // number of parameters in layer
-  int kernelpar_;     // number of parameters in layer
```

Comment:
This breaks the build because `kernelpar_` is still used below. Fixed in the current version of #47.